install.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+for file in /tmp/version/*;
+do
+	$file
```

Comment:
It seems there is some unintended indentation here, @himanshu0503 could you have a look at this please?